Makefile

```diff
@@ -26,6 +26,25 @@ GOVETTARGETS=cmd \
 	plugins \
 	utils
 
+# Setup proxies for docker build
+ifeq ($(HTTP_PROXY),)
+HTTPBUILD=
+else
+HTTPBUILD=--build-arg HTTP_PROXY=$(HTTP_PROXY)
+endif
+ifeq ($(HTTPS_PROXY),)
+HTTPSBUILD=
+else
+HTTPSBUILD=--build-arg HTTPS_PROXY=$(HTTPS_PROXY)
+endif
+
+DOCKERBUILD=docker build ${HTTPBUILD} ${HTTPSBUILD}
+
+# Test target to debug proxy issues
+checkproxy:
```

Comment:
reorder checkproxy to be after the default all target

Reply:
D'oh! Fixing now.